trestle/transforms/implementations/tanium.py

```diff
@@ -421,11 +388,14 @@ def _batch_observations(self, index: int):
                 methods=['TEST-AUTOMATED'],
                 collected=rule_use.collected
             )
-            subject_reference = SubjectReference(subject_uuid=self._get_inventory_ref(rule_use), type='inventory-item')
+            subject_uuid = self._get_inventory_ref(rule_use)
+            subject_reference = SubjectReference(subject_uuid=subject_uuid, type='inventory-item')
             observation.subjects = [subject_reference]
             observation.props = self._get_observtion_properties(rule_use)
-            observation_partial_list.append(observation)
-        return observation_partial_list
+            if subject_uuid not in observation_partial_map.keys():
+                observation_partial_map[subject_uuid] = []
```

Comment:
no need .keys() and may be able to simplify.

Reply:
Fixed.